### PR TITLE
Android

### PR DESCRIPTION
**Pull request by rizaozcelik**
Could you please merge us with the master and we can start network connection :)

### CODE REVIEW

**Comment by enescakir** [comment]
Good job Android team 👌

**Comment by rizaozcelik** [comment]
Saying good job but not merging... Whatsup bro?

File: android/Culturage/app/src/main/java/com/culturage/oceans_eleven/culturage/newsFeed/HeritageItemViewActivity.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        loc_title.setText("Any known location?");
+
+
+        TextView loc = (TextView) findViewById(R.id.her_item_location);
+        loc.setText("in the middle of nowhere");
+    }
+
+
+    private boolean retrievePost() {
+        //TODO implement api stuff
```
Comment (enescakir) [inline android/Culturage/app/src/main/java/com/culturage/oceans_eleven/culturage/newsFeed/HeritageItemViewActivity.java:67]: Don't forget to handle null values ☺️

**Review by enescakir — APPROVED** [review]
Basic Android views are added

**Pull request merged**